Java/other/RecursionCount.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+public class RecursionCount {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter first string:");
+        String str1 = scanner.nextLine();
+        System.out.println("Enter second string:");
+        String str2 = scanner.nextLine();
+        scanner.close();
+
+        System.out.println("Number of occurrences of \"" + str2 + "\" in \"" + str1 + "\":" + countSubstring(str1, str2));
+    }
+
+    private static int countSubstring(String str1, String str2) {
```

Comment:
Can you explain me, why you used access modifier with **static** keyword in method.
From my opinion, better to write only the datatype of the method with it's arguments/Signature.

Reply:
@siddharth25pandey sir, ok I will remove the `private` modifier, no particular reason behind it, just a habit of mine to keep functions private unless they are needed somewhere else.

About the `static` keyword, this function is being referenced in `main()` which is also static itself, so if I don't keep it, it will give error.